Cargo.toml

```diff
@@ -66,6 +66,3 @@ predicates = "3.0"
 pretty_assertions = "1.4"
 ring = "0.16.20"
 hex = "0.4.3"
-
-[workspace]
-members = [".", "registry"]
```

Comment:
This should be okay. The main advantage of the workspace approach is that it allows the crates to share a `target` folder. So, removing this should not break anything, just add a little bit of bloat when developing, because everything is built and cached twice.

Reply:
I think jon's trick with setting the target folder to be in the XDG_HOME or XDG_CACHE dir can help with that! That way, all projects share one target